website/docs/reference/warehouse-profiles/layer-profile.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Layer Profile"
+---
+
+:::info Vendor-supported plugin
+
+Some [core functionality](https://github.com/pingcap/dbt-tidb/blob/main/README.md#supported-features) may be limited. 
+If you're interested in contributing, check out the source code repository listed below.
+
+:::
+
+## Overview of dbt-layer
+**Maintained by:** Layer      
+**Authors:** Mehmet Ecevit  
+**Source:** [Github](https://github.com/layerai/dbt-layer)  
+**Core version:** v0.1 and newer      
+**dbt Cloud:** Not Supported      
+
+To immediately start using Layer inside your dbt DAG, install the Layer dbt Adapter for BigQuery.
+
+```
+pip install dbt-layer-bigquery -U
+```
+
+You don't need to install dbt separately. Installing `dbt-layer` will also install `dbt-core`
+
+
+### Connecting to Layer with **dbt-layer**
+
+#### Username / password authentication
+
+Add the Layer dbt Adapter to your BigQuery profile. An example profile:
+
+##### Layer connection information
+<File name='profiles.yml'>
+
+```yaml
+layer-profile:
+  target: dev
+  outputs:
+    dev:
+      type: layer_bigquery
+      method: service-account
+      project: [GCP project id]
+      dataset: [the name of your dbt dataset]
+      threads: [1 or more]
+      keyfile: [/path/to/bigquery/keyfile.json]
+      layer_project: [the Layer project to use for model training (opt)]
+      layer_api_key: [the API Key to access your Layer account (opt)]
+```
+
+</File>
+
```

Comment:
Let's put a table here to explain the fields above. See an example here:
https://docs.getdbt.com/reference/warehouse-profiles/tidb-profile#description-of-profile-fields